### PR TITLE
Fixed bug in image saver, modified gitignore, and added support for KDE

### PR DESCRIPTION
**Pull request by classPythonAddike**
I modified the image saver and downloader functions, to use os.path.join()
I also added the image directory under data/ to the gitignore. I'm not sure if this will cause issues 😉 lmk if I should revert it

Finally I added support for KDE :partying_face: and it works fine on my machine

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -140,3 +140,5 @@ cython_debug/
 # Pycharm
 .idea
 
+# image data
+data/images/*
```
Comment (Muhimen123) [inline .gitignore:144]: image

I actually had this previously in the gitignore. But faced the same problem. Currently I can think of two fixes.

1. Keep the images folder
2. check if the file exists or not. If not, then create the file(and folder) first

Second one seems better imo. You think you can handle that?

**Review by Muhimen123 — CHANGES_REQUESTED** [review]
One small fix.

**Comment by Muhimen123** [comment]
Friendly warning! I did some changes in `image_saver.py` for XFCE support. Make sure to re fork the repo to avoid conflict. 😉

**Comment by classPythonAddike** [comment]
:+1: I will work on it in a while

**Comment by classPythonAddike** [comment]
I'm not too good with github pr's, so I'll close this and create a new one with the changes :smile:

**Pull request closed without merging**